Spearhead 1944 1.1 - Mortain Map Port

Bob-Murphy opened the pull request:
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Adds port of the new SPE_Mortain Terrain

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

jaj22 commented on A3A/addons/maps/Antistasi_SPE_Mortain.SPE_Mortain/mapInfo.hpp:
```diff
@@ -0,0 +1,29 @@
+class SPE_Mortain {
+	population[] = {
+		{"Vil_Romagny",477},{"Vil_Neufbourg",562},{"Vil_Mortain",878},{"Vil_Barthelemy",305},{"Vil_Bellefontaine",453},{"Vil_Tournerie",253},{"Vil_Les_Liards",219},{"Vil_La_Vieille_Vente",78},{"Vil_La_Deliniere",209},{"Vil_La_Bagotiere",299},{"Vil_La_Baffardiere",188},{"Vil_La_Menardiere",241},{"Vil_lagranderoche",91},{"Vil_belleeau",103},{"Vil_Les_Aulnays",255},{"Vil_LaGalpichere",191},{"Vil_LeClosMon ...
```
Towns look on the large side? Might want to scale these down by 50% or so.

jaj22 commented:
**General notes**
- IIRC we don't spawn ground vehicles at the air corridor locations regardless of road access, so putting them that close just risks spawning planes on top of people.
- Six airports feels like a lot for an 8x8 km map.
- Only one of the "airports" has a runway or plane spawn locations, so there's little opportunity for rebels to fly, when that's usually one of the upsides of WW2 modsets: Rebels flying aircraft is less broken than usual.

**airport_1**
- The first two vehicle markers seem oddly placed and very oddly angled (north? surely south or east?). God knows how AIs path from these anyway.

**airport_2**
- Vehicles placed in the compound with the four vehicle placements seem incapable of pathfinding out of it and have a visceral dislike of the gate region, regardless of the gate hiding. Can't see any good reason why. Fuck Arma.
- For the other set of vehicle placements, vehicle_3 will obstruct vehicle_4 and vehicle_5.

**airport_3**
- vehicle_1 and vehicle_3 should probably be facing the road.

**airport_4**
- vehicle_1 is obstructed by vehicle. The others look ok.

**airport_5**
- Vehicles wouldn't path from either vehicle_1 or vehicle_2. vehicle_1 worked with a bit of fence-hiding but vehicle_2 is probably too tight a turn to be reliable.

**airport_6**
- I think you could build an airport with a plausible runway in the fields to the east of this one. Completely different arrangement though.
- vehicle_5 and vehicle_6 are path blocked by other vehicle spawns.
- vehicle_3 heavily overlaps the radio tower.

Bob-Murphy commented:
airport_1
- Orientation of vehicle markers adjusted
- Reordered Plane markers

airport_2
- First marker adjusted to be in front of the gate
- manually adjusted navGrid

airport_3
- Changes orientation of _1 and _3

airport_4
Again, only `airp_X_vehicle` spawns with a crew, hence no harm in other being blocked in.
- No changes due to limited space

airport_5
- Fences hidden, manually added offroad-roads in navGrid

airport_6
- Moved _6 to not collide
- Moved others to have more space and to be able to pass vehicles spawning in front of one another
- Manually added offroad-roads in navGrid

jaj22 commented:
> airport_4 Again, only `airp_X_vehicle` spawns with a crew, hence no harm in other being blocked in.

Nah, that's just the initial spawn. When QRFs spawn at a marker, they use the unoccupied vehicle places.

Bob-Murphy commented:
Moved them next to one another and hid some debris
![image](https://github.com/user-attachments/assets/c1c23ee0-3707-44f1-b19b-4130e0e0b9e5)

jaj22 reviewed (approved):
Not checked the changes.
